ui/app/AppLayouts/Communities/popups/ExportControlNodePopup.qml

```diff
@@ -0,0 +1,148 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+StatusModal {
```

Comment:
Hmm, why to avoid `StatusModal` in favour of `StatusDialog`? 🤔 

It's a `StatusDialog` and there's the convenience provided by `rightButtons` for custom buttons. I'd probably need to replace the entire footer to get access to rightButtons and customise it

Reply:
> Hmm, why to avoid StatusModal in favour of StatusDialog? :thinking:

It has annoying issues: https://github.com/status-im/status-desktop/issues/7556. `StatusDialog` was introduced and `StatusModal` was marked as deprecated in an attempt to gradually eliminate its usage from our codebase (a process that has not yet been fully realized). The rule is to use `StatusDialog` for new code.

> It's a StatusDialog and there's the convenience provided by rightButtons for custom buttons. I'd probably need to replace the entire footer to get access to rightButtons and customise it

https://github.com/status-im/status-desktop/blob/db363b378236cf0b1a69c39e746ee2d20db28891/ui/StatusQ/sandbox/pages/StatusDialogPage.qml#L204-L219

> It was marked as deprecated but perhaps it's no longer an issue now that it's based on StatusDialog

Even if it's functioning correctly at the moment (which I'm skeptical about), we should maintain only one of them to avoid any confusion about which one to use. I would recommend keeping it as deprecated and remain committed to the objective of its removal.

Reply:
Thanks for the explanation! 🍻
I'll update it

Reply:
Done!